emails/tests/fixtures/README.md

```diff
@@ -1,10 +1,21 @@
-## Email test fixtures
+# Email test fixtures
 
 These fixtures are used in `emails/tests/views_tests.py`
 
-### Incoming Emails
+## Incoming Emails
 
-Some fixtures capture the [SNS][] Delivery [notification JSON][] representing an
+Incoming Email fixtures come in two varieties. The original fixtures capture the [SNS][]
+Delivery [notification JSON][] representing an incoming email. The newer fixtures are
+the raw incoming emails.
+
+Both fixtures have related fixtures for the expected output email.
```

Comment:
suggestion (non-blocking): I didn't understand this at first glance. Maybe this?
```suggestion
Every incoming email fixture has a corresponding fixture for the expected output email.
```